examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter

```diff
@@ -0,0 +1,3546 @@
+    ram      attribute clusterRevision default = 4;
+
+    handle command AddGroup;
+    handle command AddGroupResponse;
+    handle command ViewGroup;
+    handle command ViewGroupResponse;
+    handle command GetGroupMembership;
+    handle command GetGroupMembershipResponse;
+    handle command RemoveGroupResponse;
+    handle command RemoveGroup;
+    handle command RemoveAllGroups;
+    handle command AddGroupIfIdentifying;
+  }
+
+  server cluster Descriptor {
+    callback attribute deviceTypeList;
+    callback attribute serverList;
+    callback attribute clientList;
+    callback attribute partsList;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
+    callback attribute featureMap;
+    callback attribute clusterRevision;
+  }
+
+  server cluster IlluminanceMeasurement {
+    ram      attribute measuredValue default = 0xC351;
+    ram      attribute minMeasuredValue default = 1;
+    ram      attribute maxMeasuredValue default = 0xfffe;
+    ram      attribute lightSensorType default = 1;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 2;
+  }
+}
+endpoint 3 {
+  device type ma_occupancysensor = 263, version 4;
+
+  binding cluster Binding;
+
+  server cluster Identify {
+    ram      attribute identifyTime default = 0x0;
+    ram      attribute identifyType default = 0x0;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 2;
+
+    handle command Identify;
+    handle command TriggerEffect;
+  }
+
+  server cluster Groups {
+    ram      attribute nameSupport;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 4;
+
+    handle command AddGroup;
+    handle command AddGroupResponse;
+    handle command ViewGroup;
+    handle command ViewGroupResponse;
+    handle command GetGroupMembershipResponse;
+    handle command GetGroupMembership;
+    handle command RemoveGroupResponse;
+    handle command RemoveGroup;
+    handle command RemoveAllGroups;
+    handle command AddGroupIfIdentifying;
+  }
+
+  server cluster Descriptor {
+    callback attribute deviceTypeList;
+    callback attribute serverList;
+    callback attribute clientList;
+    callback attribute partsList;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
+    callback attribute featureMap;
+    callback attribute clusterRevision;
+  }
+
+  server cluster OccupancySensing {
```

Comment:
Please run TC_DeviceBasicComposition.py and TC_DeviceConformance.py against the device to resolve issues, and ensure everything is up to latest.

Reply:
Done

* TC_DeviceBasicComposition.py
```
(py) $ python ./src/python_testing/TC_DeviceBasicComposition.py  --storage-path admin_storage.json --manual-code 34970112332 --bool-arg ignore_in_progress:True allow_provisional:True --PICS src/app/tests/suites/certification/ci-pics-values
...
[MatterTest] 11-12 21:47:08.492 INFO ###########################################################
[MatterTest] 11-12 21:47:08.492 INFO Summary for test class TC_DeviceBasicComposition: Error 0, Executed 8, Failed 0, Passed 8, Requested 14, Skipped 6
...
INFO:root:Final result: PASS !
```

* TC_DeviceConformance.py
```
$ python ./src/python_testing/TC_DeviceConformance.py  --storage-path admin_storage.json --manual-code 34970112332 --bool-arg ignore_in_progress:True allow_provisional:True --PICS src/app/tests/suites/certification/ci-pics-values
...
[MatterTest] 11-13 19:24:17.361 INFO Summary for test class TC_DeviceConformance: Error 0, Executed 3, Failed 0, Passed 3, Requested 3, Skipped 0
[MatterTest] 11-13 19:24:17.362 INFO Summary for test run MatterTest@11-13-2024_19-24-11-654:
Total time elapsed 5.707147184992209s
Artifacts are saved in "/tmp/matter_testing/logs/MatterTest/11-13-2024_19-24-11-654"
Test summary saved in "/tmp/matter_testing/logs/MatterTest/11-13-2024_19-24-11-654/test_summary.yaml"
Test results: Error 0, Executed 3, Failed 0, Passed 3, Requested 3, Skipped 0
...
INFO:root:Final result: PASS !
```